query-engine/connector-test-kit/src/test/scala/util/TestServer.scala

```diff
@@ -99,78 +99,46 @@ case class TestServer() extends PlayJsonExtensions with LogSupport {
     val binaryLogLevel   = "RUST_LOG" -> s"query_engine=$logLevel,quaint=$logLevel,query_core=$logLevel,query_connector=$logLevel,sql_query_connector=$logLevel,prisma_models=$logLevel,sql_introspection_connector=$logLevel"
     val log_requests_env = if (log_requests) { "LOG_QUERIES" -> "y" } else { ("", "") }
 
-    val response = (project.isPgBouncer, legacy) match {
```

Comment:
the setting for pg bouncer is suddenly not needed anymore?

Reply:
Yeah. It's now part of the connection string. I should take this flag out completely.